base/int.jl

```diff
@@ -553,14 +553,66 @@ floor(::Type{T}, x::Integer) where {T<:Integer} = convert(T, x)
 
 ## integer construction ##
 
+"""
+    @int128_str str
+
+Parse `str` as an [`Int128`](@ref).
+
+# Examples
+```jldoctest; filter = r"Stacktrace:(\\n \\[[0-9]+\\].*)*"
+julia> @int128_str "123456789123"
+123456789123
+
+julia> @int128_str "123456789123.4"
```

Comment:
Use the `int128"1234.4"` syntax here and in the other examples since that is how they are used in 99% of the cases?